docs/server/source/quickstart.md

```diff
@@ -1,6 +1,12 @@
 # Quickstart
 
-This page has instructions to set up a single stand-alone BigchainDB node for learning or experimenting. Instructions for other cases are [elsewhere](introduction.html). We will assume you're using Ubuntu 16.04 or similar. If you're not using Linux, then you might try [running BigchainDB with Docker](appendices/run-with-docker.html).
+This page has instructions to set up a single stand-alone BigchainDB node manually for learning or experimenting. Instructions for other cases are [elsewhere](introduction.html). You might try one of the following deployment methods
+as well, depending on your choice:
+- [BigchainDB with Docker](appendices/run-with-docker.html).
+- [BigchainDB with Vagrant](appendices/run-with-vagrant.html).
+- [BigchainDB with Ansible](appendices/run-with-ansible.html).
+
+For manual installation, We will assume you're using Ubuntu 16.04 or similar.
```

Comment:
Maybe remove this line and embed the "Ubuntu 16.04" in the opening line, e.g.

This page has instructions to set up a single stand-alone BigchainDB node on Ubuntu 16.04, for learning or experimenting.

I removed the word "manually" because even this method uses helper tools such as `apt` and `pip` (but not Ansible or Vagrant). Basically, it's not clear what the word "manually" means in the context of installing software, so let's just avoid it.

Reply:
If I don't specify manually both the deployment methods will be the same and manually implies the opposite of `automat(ic/ed)`. All deployments need some manual intervention or pre-req installations etc.

Reply:
If you want to keep the word "manually" then that is fine.

Reply:
"We" => "we"